src/renderer/components/+workloads-pods/details/volumes/variants/projected.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import React from "react";
+import { DrawerItem, DrawerTitle } from "../../../../drawer";
+import type { VolumeVariantComponent } from "../variant-helpers";
+
+export const Projected: VolumeVariantComponent<"projected"> = (
+  ({ variant: { sources, defaultMode }}) => (
+    <>
+      <DrawerItem name="Default Mount Mode">
+          0o{defaultMode.toString(8)}
+      </DrawerItem>
+      <DrawerItem name="Sources">
+        {
+          sources.map(({ secret, downwardAPI, configMap, serviceAccountToken }, index) => (
+            <React.Fragment key={index}>
+              {secret && (
+                <>
+                  <DrawerTitle size="sub-title">Secret</DrawerTitle>
+                  <DrawerItem name="Name">
+                    {secret.name}
+                  </DrawerItem>
+                  <DrawerItem name="Items">
+                    <ul>
+                      {secret.items?.map(({ key }) => <li key={key}>{key}</li>)}
```

Comment:
why use `key` here but `path` for downwardAPI and configMap? Other fields could be shown too? Or it's overkill?

Reply:
So for `downwardAPI` there is no `key` field so it is not possible. However, on further investigation I admit that the display for `secret` and `configMap` it could be better